src/App.js

```diff
@@ -1,10 +1,32 @@
 import "./styles.css";
+import { Route, Routes } from "react-router-dom";
+import NavDrawer from "./components/NavDrawer";
+import Home from "./pages/Home/";
+import Login from "./pages/Login";
+import Signup from "./pages/Signup";
+import Playlists from "./pages/Playlists";
+import SinglePlaylist from "./pages/SinglePlaylist";
```

Comment:
Barrel export will help here

Reply:
Thanks for the suggestion, instead I'll separate the routes to make the App.js cleaner.